config/enterprise_versions.yml

```diff
@@ -57,6 +58,9 @@ components:
   intrusion-detection-controller:
     image: tigera/intrusion-detection-controller
     version: master
+  anomaly-detection-jobs:
```

Comment:
This needs to be `anomaly_detection_jobs` to match what's in the process repo (which builds our hash releases).